app/controllers/well_known/keybase_proof_config_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module WellKnown
+  class KeybaseProofConfigController < ActionController::Base
+    before_action :set_default_response_format
+
+    def show
+      # see https://keybase.io/docs/proof_integration_guide#1-config
+      # specific to each mastodon instance
+      @my_domain = ExternalProofService.my_domain
+      @domain_display = ExternalProofService.my_domain_displayed
+      @contacts = ExternalProofService::Keybase.my_contacts
+      @svg_black = 'https://glsk.net/wp-content/themes/glsk_minimal_en/images/mastodon.svg'
+      @svg_full = 'https://upload.wikimedia.org/wikipedia/commons/4/48/Mastodon_Logotype_%28Simple%29.svg'
```

Comment:
We have copies of this logo and the one above in the `app/javascript/images` directory. URL can be generated with `full_asset_url(asset_path('logo.svg'))` etc

Reply:
But also I believe those definitions would go into the serializer, not the controller.